src/code/shrink_window.c

```diff
@@ -0,0 +1,99 @@
+#include <global.h>
+
+s32 D_8012CED0 = 0;
+
+s32 sShrinkWindowVal = 0;
+
+s32 sShrinkWindowNowVal = 0;
+
+void ShrinkWindow_SetVal(s32 value) {
+
+    if (HREG(80) == 0x13 && HREG(81) == 1) {
+        osSyncPrintf("shrink_window_setval(%d)\n", value);
+    }
+    sShrinkWindowVal = value;
+}
+
+u32 ShrinkWindow_GetVal(void) {
+    return sShrinkWindowVal;
+}
+
+void ShrinkWindow_SetNowVal(s32 nowVal) {
+    if (HREG(80) == 0x13 && HREG(81) == 1) {
+        osSyncPrintf("shrink_window_setnowval(%d)\n", nowVal);
+    }
```

Comment:
NowVal sounds like a oot dev-ism and not like a good name that we should really keep , if you know what i mean. 
sounds kind of like "magic" "magic_now" and "magic_now_now" lmao

if theres no good name for it then oh well, but if theres something else that makes sense i think itd be good to call it something other than the og name

Reply:
I think "now" is basically what they used for "current", but I agree we shouldn't use it if possible.